src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -31,6 +31,7 @@
 var configData = require('./library_service_client_config');
 var extend = require('extend');
 var gax = require('google-gax');
+var _ = require('lodash');
```

Comment:
better to use 'lodash.merge' module to minimize the dependency size: https://www.npmjs.com/package/lodash.merge/